docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,81 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters and configure the `with_geometries` attribute accordingly:
```

Comment:
```suggestion
To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without frame system configuration.
Configure the `with_geometries` attribute accordingly:
```
@npentrel Suggestion re. your comment about intrinsic parameters: (1/2) (other one below)
I think this is mostly what is meant and might help to avoid confusion

Reply:
oh so it's not intrinsic parameters? or are those the same?